share/pot/vnet-start.sh

```diff
@@ -91,22 +91,23 @@ _ipv4_start()
 	(
 		echo "ext_if = \"${POT_EXTIF}\""
 		echo "localnet = \"${POT_NETWORK}\""
-		echo "nat on \$ext_if from \$localnet to any -> (\$ext_if)"
+		echo "nat on \$ext_if from \$localnet to any -> (\$ext_if:0)"
```

Comment:
It would be kind of cool if it was possible to configure the IP address to NAT on (as it isn't always the first one). Future feature probably :)

Reply:
yes, this is the plan.
I wanted to unlock the issue with alias on DigitalOcean, but it makes totally sense to be more flexible and to provide a way to specify which address you want to use.